src/lsp-server.ts

```diff
@@ -655,14 +656,15 @@ export class LspServer {
                 offset: params.position.character + 1,
                 triggerCharacter: getCompletionTriggerCharacter(params.context?.triggerCharacter),
                 triggerKind: params.context?.triggerKind,
+                includeInsertTextCompletions: true,
```

Comment:
Do you know if this is still relevant if client does not support InsertReplace? Maybe we should make it conditional on the capability if that will make us save some cycles and bytes?

Reply:
this is probably not necessary and `includeInsertTextCompletions` is deprecated, 
I tried everything just to make this work :D and forgot to remove this.